extensions/remove-paywall/package.json

```
@@ -68,12 +73,16 @@
         {
           "title": "RemovePaywall (removepaywall.com)",
           "value": "https://www.removepaywall.com"
+        },
+        {
+          "title": "Freedium.cfd (freedium.cfd)",
```

Just nitpicking here: if the service name is "Freedium", omit the `.cfd` suffix in its name

```suggestion
          "title": "Freedium (freedium.cfd)",
```

While you're there, can you also make 12ft consistent with the naming?

```
- "title": "12ft.io",
+ "title": "12ft (12ft.io)",
```

Thanks!